plasma_framework/package.json

```diff
@@ -14,10 +14,11 @@
     "solidoc": "^1.0.5"
   },
   "devDependencies": {
-    "elliptic": ">=6.5.3",
     "@codechecks/client": "^0.1.10",
+    "@openzeppelin/test-helpers": "^0.5.6",
```

Comment:
The older version of the test helpers package could be uninstalled

Reply:
Thanks, I decided to stay with the old version and remove the new one. I noticed I didn't use the new version.

Reply:
@pgebal Can we do a clean uninstall then? looks like it's still there in package-lock